<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: white;
    border-radius: 20px;
}

QPushButton
{
    color: white;
    background-color: #6357B7;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
}

QPushButton:Pressed
{
    background-color: #C397E4;
}

QPushButton:disabled
{
    color: #6357B7;
    background-color: white;
    border-radius: 0px;
}

QLineEdit
{
    color: #6357B7;
    background-color: white;
    border : 2px solid #6357B7;
    border-radius: 15px;
    font: 10pt &quot;Bahnschrift&quot;;
    border: 1px solid #6357B7;
    padding: 10px;
}

QLabel
{
    background-color: transparent;
    color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}

QComboBox
{
    border: 1px solid #6357B7;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
	color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}

QTextEdit
{
    color: #6357B7;
    background-color: white;
    border : 2px solid #6357B7;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
    border: 1px solid #6357B7;
    padding: 13px;
}

QTableWidget
{
	color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Ingresar Avion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Tipo de Propulsion</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>136</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>136</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Peso Nominal</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Numero de Motores</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>135</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Modelo</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Area</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Capacidad</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Descripcion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Ingresar Avion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Aviones Registrados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>68</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Id Avion</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Modelo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Capacidad</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>43</height>
         </size>
        </property>
        <property name="text">
         <string>Eliminar Avion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
